components/collapsible/collapsible.vue

```diff
@@ -229,14 +227,7 @@ export default {
     labelledBy () {
       // aria-labelledby should be set only if aria-labelledby is passed as a prop, or if
       // there is no aria-label and the labelledby should point to the anchor
-      return this.ariaLabelledby || (!this.ariaLabel && getUniqueString('DtCollapsible__anchor'));
-    },
-
-    contentStyle () {
```

Comment:
I deleted this because we are not using it.
Also we are not using it in the current staging-vue3 branch [check](https://github.com/dialpad/dialtone-vue/blob/staging-vue3/components/collapsible/collapsible.vue)

Reply:
Doesn't seem needed.. you can remove